src/Campaigns/Blocks/shared/hooks/useCampaign.ts

```diff
@@ -0,0 +1,11 @@
+import {useEntityRecord} from '@wordpress/core-data';
+import {Campaign} from '@givewp/campaigns/admin/components/types';
+
+export default function useCampaign(campaignId) {
```

Comment:
Can we add the type for `campaignId`?

Reply:
Resolved: a6aa8de7c766c3e25adf7f3d1dd8b8232d94fdc6